### ISSUE TITLE
Syberia ultrawide fix problem :( 

### ISSUE THREAD

**Issue by crazyivanpl** [Lyall/SyberiaTWBFix#2]
Hi sorry for bad english ;) 

the game works at a resolution of 3440x1440 as it should, but the use of a graphics card is a maximum of 45-55% I do not know why :( Ryzen 5600 + rtx 3080ti.  Original game on Steam :)  reinstall dont solve this problem :(

**Comment by Lyall** [comment]
Hey,

Thank you for reporting this. I went through the code for the mod and it was indeed causing performance problems. I didn't see it on my end as I was capped to 60. I have since gone and rewritten the resolution patching portion of the mod. You can try it out now in [release v1.0.2](https://github.com/Lyall/SyberiaTWBFix/releases/tag/v1.0.2). Let me know if it works for you and I'll close the issue.

TL;DR it should be fixed in the new version.

For reference here is a comparison of how it ran vanilla with no mod, then with v1.0.1 and finally with the new v1.0.2.
![sybTWB 2560x1440 vanilla](https://user-images.githubusercontent.com/695941/159622747-43850584-ebb8-426e-985a-a6532194e03f.jpg)
![sybTWB 2560x1440 mod v1 0 1](https://user-images.githubusercontent.com/695941/159622756-700c58dc-3244-4e16-ab0d-9283b37baf4a.jpg)
![sybTWB 2560x1440 mod v1 0 2](https://user-images.githubusercontent.com/695941/159622758-fdec43cd-67a2-459a-b23a-e8ee5667efb5.jpg)

**Comment by crazyivanpl** [comment]
Hi :) 
now the use of graphics card is 99% plus good fps :)

Thanks a lot for your help and the whole ultrawide fix :) great job :)

Regards :)

**Comment by Lyall** [comment]
That's great to hear. Enjoy the game! I'll close the issue for now.
